tests/playwright/src/model/pages/create-kind-cluster-page.ts

```diff
@@ -106,6 +106,7 @@ export class CreateKindClusterPage extends BasePage {
   private async createCluster(timeout: number = 120000): Promise<void> {
     await playExpect(this.clusterCreationButton).toBeVisible();
     await this.clusterCreationButton.click();
+    await this.logsButton.click();
```

Comment:
maybe also add `scrollIntoView` function which will move the focus no the element.

Reply:
For some reason it doesn't work
```
TimeoutError: locator.scrollIntoViewIfNeeded: Timeout 30000ms exceeded.
Call log:
  - waiting for getByRole('form', { name: 'Properties Information' }).getByRole('button', { name: 'Show Logs' })
```
But it works fine without scrolling :)